backstop/tests/carousel.html

```diff
@@ -0,0 +1,529 @@
+            id="demo-slide-4"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="4"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="5 of 11"
+            aria-hidden="true"
+            id="demo-slide-5"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="5"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="6 of 11"
+            aria-hidden="true"
+            id="demo-slide-6"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="6"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="7 of 11"
+            aria-hidden="true"
+            id="demo-slide-7"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="7"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="8 of 11"
+            aria-hidden="true"
+            id="demo-slide-8"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="8"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="9 of 11"
+            aria-hidden="true"
+            id="demo-slide-9"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="9"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="10 of 11"
+            aria-hidden="true"
+            id="demo-slide-10"
+          ></li>
+
+          <li
+            class="iui-carousel-slider-item demo-carousel-height"
+            data-slide-id="10"
+            role="tabpanel"
+            aria-roledescription="slide"
+            aria-label="11 of 11"
+            aria-hidden="true"
+            id="demo-slide-11"
+          ></li>
+
+        </ol>
+        <nav class="iui-carousel-navigation">
+          <div class="iui-carousel-navigation-left">
+            <button
+              class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+              type="button"
+              aria-controls="myCarousel-items"
+              aria-label="Previous slide"
+              tabindex="-1"
+            >
+              <svg-chevron-left
+                class="iui-button-icon"
+                aria-hidden="true"
+              ></svg-chevron-left>
+            </button>
+            <button
+              class="iui-button iui-borderless iui-small iui-carousel-navigation-animation-button"
+              type="button"
+              aria-label="Pause animation"
+              data-action="pause"
+              tabindex="-1"
```

Comment:
Is this supposed to be not focusable now?
If no, we have those rules with target of `data-pressed`. Dont need for animation button?

Reply:
Instead of focus going to each individual button we now have focus going on the entire carousel.  From there the user can use the left / right arrow keys or the spacebar / enter key to play / pause.  Unsure about the `data-pressed` thing.